kubernetes/argocd/stacks/traefik/values.yml

```diff
@@ -13,3 +13,7 @@ tlsStore:
   default:
     defaultCertificate:
       secretName: default-cert
+ports:
+  web:
+    redirectTo:
+      port: websecure
```

Comment:
This definitely isn't valid.
These are values passed into the Helm chart which don't call for a `redirectTo` attribute here.

Reply:
Sorry, I retract my previous statement on this.
I was looking for a `.Values.ports.web.redirectTo` variable used in the Helm chart, however it that isn't ever used in the Helm chart. 
Discussed on Discord and it was mentioned that the syntax is from the Helm chart values.yaml file itself, so this is probably correct.

```
    # Port Redirections
    # Added in 2.2, you can make permanent redirects via entrypoints.
    # https://docs.traefik.io/routing/entrypoints/#redirection
    # redirectTo: websecure
```

Reply:
It's important to note that we currently use the `23.1.0` Helm chart version, which has a slightly different syntax than the latest.
![image](https://github.com/acm-uic/IaC/assets/1387910/0419025f-99ad-4bd6-8c1a-f8520bce39ac)

We should either just change it to the syntax used in the `23.1.0` chart, or upgrade and make the corresponding changes in the `values.yml`.

Reply:
Fixed in newest commit